resources/views/home.blade.php

```diff
@@ -83,6 +83,14 @@ function setColorMode(dark) {
                                 </label>
                             </div>
                         @endforeach
+                        <label for="font_size" class="form-label fw-bold mt-4">Font Size</label>
+                        <select name="font_size" id="font_size" class="form-select">
```

Comment:
maybe as a follow-up later, use the same laravel syntax for selects here that we use on line 63?

https://github.com/code4recovery/pdf/pull/82/files#diff-866658882d8457254ffdc08e724486f1677695c8e4dd25d6994dd6291b63f2f0R63

Reply:
Thanks @joshreisner! Missed that. @SIA-WEBSUP I can pull and edit the syntax if you prefer.

Reply:
Please do... I'm away from home for the next few days.
Thank you Both.